docs/book/src/topics/managedcluster.md

```diff
@@ -175,6 +175,29 @@ should be fairly clear from context.
 | networkPolicy             | azure, calico                 |
 
 
+### Use an existing Virtual Network to provision an AKS cluster.
+
+If you already have an existing VNet and Subnet, you can deploy the AKS cluster within the VNet by configuring the AzureManagedControlPlane object in the following manner.
```

Comment:
Can we change this to something like:

"If you'd like to deploy your AKS cluster in an existing Virtual Network, but create the cluster itself in a different resource group, you can configure the `AzureManagedControlPlane` resource with a reference to the existing Virtual Network and subnet. For example:

"

Something like that

Reply:
ack